tests/jobs/scala/src/main/scala/KafkaCassandraStream.scala

```diff
@@ -0,0 +1,137 @@
+import java.util.UUID.randomUUID
+
+import com.datastax.spark.connector._
+import com.datastax.spark.connector.cql.CassandraConnector
+import org.apache.spark.mllib.random.RandomRDDs.normalRDD
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.streaming.dstream.ConstantInputDStream
+import org.apache.spark.streaming.{Seconds, StreamingContext}
+import scopt.OptionParser
+
+import scala.math.max
+import scala.util.Random
+
+
+case class Employee(
+                     id: String,
+                     first_name: String,
+                     last_name: String,
+                     email_address: String,
+                     address: String,
+                     city: String
+                   )
+
+/**
+  * Application that generates random records and streams them to Cassandra.
+  *
+  * Usage:
+  * {SPARK} {ARGS} {PATH_TO_ASSEMBLY}
+  * where {SPARK} is either `dcos spark run` or `./bin/spark-submit`
+  * {ARGS} are the configuration arguments to Spark.
+  * {PATH_TO_ASSEMBLY} is the location of the snapshot jar containing this application
+  *
+  * Example:
+  * dcos spark run --submit-args='\
+  * --conf spark.cassandra.connection.host=node-0-server.cassandra.autoip.dcos.thisdcos.directory \
+  * --conf spark.cassandra.connection.port=9042 \
+  * --conf spark.mesos.executor.docker.image=mesosphere/spark:latest \
+  * --conf spark.mesos.executor.home=/opt/spark/dist \
+  * --class KafkaCassandraStream \
+  * http://infinity-artifacts.s3.amazonaws.com/autodelete7d/dcos-spark-scala-tests-assembly-0.1-SNAPSHOT.jar \
+  * --numberOfRecords 100000 --numberOfPartitions 10 --minColumnsLength 20 --maxColumnsLength 30'
+  */
+object KafkaCassandraStream {
```

Comment:
This class is useful for testing writing to Cassandra (the original intent of SPARK-671), but does not interact with Kafka. My proposal would be to drop it (and the associated integration test) from this PR and create a new PR that only addresses the Spark->Cassandra interaction. This was definitely my bad for hi-jacking the PR to add the Kafka->Spark->Cassandra wordcount example. Sorry.